py3status/modules/volume_status.py

```diff
@@ -149,6 +143,97 @@ def current_volume(self):
         # get info about channel mute status
         muted = self._get_muted(output)
 
+        return perc, muted
+
+    def volume_up(self, delta):
+        # volume up
+        cmd = 'amixer -q -D {} sset {} '.format(self.device, self.channel)
```

Comment:
Done. I open it in the `__init__` of the base class. I didn't do it at first because I try to open files as little as possible.